tests/unit/lib/NavLink-test.js

```diff
@@ -101,6 +101,15 @@ describe('NavLink', function () {
             );
             expect(link.getDOMNode().getAttribute('class')).to.equal('active');
         });
+        it('should set active state by tag name if the optional activeElement property is set', function () {
+            var navParams = {a: 1, b: 2};
+            var link = ReactTestUtils.renderIntoDocument(
+                <MockAppComponent context={mockContext}>
+                    <NavLink activeElement="span" routeName='foo' />
+                </MockAppComponent>
+            );
+            expect(link.getDOMNode().nodeName.toLowerCase()).to.equal('span');
```

Comment:
That would apply to all tests, not my pull request in particular. I like to be consistent. :)